shared/actions/wallets.js

```diff
@@ -262,12 +262,13 @@ const doRefreshPayments = (state: TypedState, action: WalletsGen.RefreshPayments
     const {accountID, paymentID} = action.payload
     const paymentsReceived = createPaymentsReceived(action.payload.accountID, payments, pending)
     const found =
-          paymentsReceived.payload.payments.find(elem => elem.id === paymentID) || paymentsReceived.payload.pending.find(elem => elem.id === paymentID)
+      paymentsReceived.payload.payments.find(elem => elem.id === paymentID) ||
```

Comment:
Huh, I guess prettier did this -- @akalin-keybase, any idea which of us has prettier misconfigured?

Reply:
could have been me -- my pre-commit was borked on my (newly-reinstalled) laptop.

CI would catch this ideally, though...